src/logging.ts

```diff
@@ -0,0 +1,52 @@
+import { EventEmitter } from "events";
```

Comment:
I ended up dissatisfied with all the logging libraries I tried, so landed on the simple wrapper you see here. The idea behind is being an `EventEmitter` is something else can subscribe to to those events, filter if desired, batch them up, and send them over the websocket to use every so often. This just lays the groundwork for that.